presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreenStateHolder.kt

```diff
@@ -31,14 +33,15 @@ import java.util.*
 
 @Stable
 class LockScreenStateHolder(
-    context: Context,
+    val context: Context,
```

Comment:
```suggestion
    private val context: Context,
```

private로 해도 될 것 같아요!

Reply:
@jaeryo2357 기존처럼 다시 변경했습니다 [5bf2eaf](https://github.com/Knock-Lock/Knock-Lock/pull/95/commits/5bf2eaf88e73c2a0ec0f7b7ad72098cb78bab70b)